src/pages/VenuePage/VenuePage.tsx

```diff
@@ -38,25 +42,105 @@ import { useVenueId } from "hooks/useVenueId";
 // import { useVenueAccess } from "hooks/useVenueAccess";
 import useConnectCurrentVenue from "hooks/useConnectCurrentVenue";
 
-import { CountDown } from "components/molecules/CountDown";
-import { LoadingPage } from "components/molecules/LoadingPage/LoadingPage";
-// import { AccessDeniedModal } from "components/atoms/AccessDeniedModal/AccessDeniedModal";
-import TemplateWrapper from "./TemplateWrapper";
-
 import { updateTheme } from "./helpers";
 
+import Login from "pages/Account/Login";
+
 import "./VenuePage.scss";
 
-import Login from "pages/Account/Login";
-import { useRelatedVenues } from "hooks/useRelatedVenues";
+const CountDown = React.lazy(() =>
+  import("components/molecules/CountDown").then((m) => ({
+    default: m.CountDown,
+  }))
+);
+const LoadingPage = React.lazy(() =>
+  import("components/molecules/LoadingPage/LoadingPage").then((m) => ({
+    default: m.LoadingPage,
+  }))
+);
+const TemplateWrapper = React.lazy(() => import("./TemplateWrapper"));
```

Comment:
Lazy loading `LoadingPage` doesn't seem to make a lot of sense to me..?

Also, as per previous comment, this should probably be reverted due to conflicts with:

- https://github.com/sparkletown/sparkle/pull/1737

Reply:
Removed lazy load in b7ec26dc13ee3c39f4d061fe3565ff7cd02b0f3e